flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAddPartitions.java

```diff
@@ -78,8 +78,8 @@ public List<SqlNode> getOperandList() {
     }
 
     @Override
```

Comment:
We should remove the `@Override` as well as the method. This occurs else where also